bucket/hwp-converter.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "2.0",
+    "description": "A tool which allows you to convert and open HWP files in Word as DOCX file",
+    "homepage": "https://www.microsoft.com/en-us/download/details.aspx?id=49152",
+    "license": "Proprietary",
+    "architecture": {
+        "32bit": {
+            "url": "https://download.microsoft.com/download/1/8/D/18D83826-2F32-4535-8C6A-686149B39DF8/hwpconverter_x86_en-us.exe#dl.7z",
+            "hash": "48d599393a0a15b9c81a453bfa2e8db5d546e4cb63045cc382629f4fa35db82e",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\hwpconverter.msi\" -DestinationPath \"$dir\\tmp\"",
```

Comment:
7zip Archive?

Reply:
I used it to extract executable files from msi and it works. Any other alternative is welcome.